src/Microsoft.AspNet.Mvc.Core/ModelBinding/Validation/ValidationVisitor.cs

```diff
@@ -191,11 +182,16 @@ private bool VisitEnumerableType()
         {
             var isValid = true;
 
-            if (_model != null)
+            if (_model != null && _metadata.ValidateChildren)
             {
                 var strategy = _strategy ?? DefaultCollectionValidationStrategy.Instance;
                 isValid = VisitChildren(strategy);
             }
+            else if (_model != null)
+            {
+                // Suppress validation for the prefix, but we still want to validate the object.
```

Comment:
If it does set the model's entry to skipped, then the subsequent code sets it back to valid. The end result is that the node for `_model` gets validated.